src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -1185,18 +1176,14 @@ namespace HVACDXHeatPumpSystem {
         // PURPOSE OF THIS SUBROUTINE:
         // Get inlet node number
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int NodeNum;
-        int DXHeatSysNum;
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:1190]:(style),[variableScope],The scope of the variable 'DXHeatSysNum' can be reduced.